akka-cluster-sharding-typed/src/test/scala/akka/cluster/sharding/typed/JoinConfigCompatCheckerClusterShardingSpec.scala

```diff
@@ -21,6 +26,7 @@ object JoinConfig {
 
   val baseConfig: Config =
     ConfigFactory.parseString("""
+      akka.loggers = ["akka.event.slf4j.Slf4jLogger"]
```

Comment:
isn't this added automatically?

Reply:
Not sure why but here it is not

Reply:
As in, without it some of the logging would be captured but some warnings about dead letters and connection tear down would leak through

Reply:
strange, I'd like to look into why that is the case

Reply:
The reason is that the `akka.testkit.TestEventListener` via

```
.withFallback(AkkaSpec.testConf)
```

meaning that both TestEventListener and Slf4jLogger are used. The output is from TestEventListener.

Remove that fallback and it will work without setting the akka.loggers here.

Reply:
Ah, thanks for figuring that out for me.